user_test.go

```diff
@@ -0,0 +1,57 @@
+package workwx
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"testing"
+)
+
+var once sync.Once
+var weClient *WorkwxApp
+
+func getWorkWeClient() *WorkwxApp {
+	corpID := "your_corpid"
+	corpSecret := "your_corpsecret"
+	agentID := int64(1234567)
```

Comment:
Remove the E2E tests (and consequently entirety of this file), the secrets are fake so the CI definitely won't pass.

Unfortunately until I (or someone kind enough) set up some secrets myself tests can only be done locally... I personally maintain the `workwxctl` utility along with any API change so I get to test with `workwxctl` from time to time. The E2E tests are harder to set up than you might think, IIRC you have to own a company before getting a Work Wechat account to play with. After you *actually* set up a company you better do some *real* things rather than just using the account as a test sandbox...

You can test with your own staging environment along with rest of your business logic for the time being.